docs/sphinx/Physics/DiskModel.md

```diff
@@ -0,0 +1,323 @@
+# Disk Model
+## Overview
+
+One of Cholla's flagship applications is Idealized Galaxy Simulations of disk galaxies.
+These simulations can involve choices related to several physical concepts: (i) modeling of gravity, (ii) radiative cooling, and (iii) modeling stellar feedback. These modules are all highly related.
+You also need to consider 2 simulation phases: initial-conditions and evolution over a timestep.
+
+### Primer on Gravitational Potentials
+
+A central ingredient of any galaxy simulation is the physical model of the gravitational potential. This is a complex topic that is relevant in the context of initial conditions **AND** in the context of evolving the simulation over a timestep. The precise treatment of the potential depends on the context **AND** on Cholla's configuration.
+
+It is instructive to first define all possible components of the total potential. Throughout the rest of this page, we will subsequently describe how treatment of these potentials can vary. The total potential is given by: 
+
+:::{math}
+
+\Phi_{\rm tot} = \Phi_{\rm dm} + \Phi_{\rm stars,old} + \Phi_{\rm stars,young} + \Phi_{\rm gas}
+:::
+
+where
+-  {math}`\Phi_{\rm dm}` corresponds to the dark matter halo  (always a static potential)
+-  {math}`\Phi_{\rm stars,old}` corresponds to older stellar populations in the disk (always a static potential)
+- {math}`\Phi_{\rm stars,young}` corresponds to the younger stellar populations (computed with self-gravity from particles)
+- {math}`\Phi_{\rm gas}` corresponds to the younger stellar populations (computed with self-gravity from particles)
+
+### Primer on Radiative Cooling
+
+At the time of writing, simulations must always be initialized with CIE cooling (cooling shuts off below 1e4 K) in order to prevent immediate of the gas disk. The current strategy for using other cooling mechanisms. After running the simulation long enough to start driving turbulence in the gas disk (commonly a few 10s of Myr), you can then restart the simulation with a different cooling recipe.
+
+### Primer on Feedback and Star Formation:
+
+In a simulation without particles, we feedback is implemented with a CGOLs like mechanism.
```

Comment:
This option is not actually included in this version of cholla, right? My recollection is that the CGOLS feedback code has only ever existed in branches that have not been merged into this repo, in which case, maybe we should not mention it here?